backend/app/transcendence/test/app.e2e-spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { AppModule } from "./../src/app.module";
+
+describe("AppController (e2e)", () =>
+{
+	let app: INestApplication;
+
+	beforeEach(async () =>
+	{
+		const moduleFixture: TestingModule = await Test.createTestingModule(
+		{
+			imports: [AppModule],
+		})
+		.compile();
+
+		app = moduleFixture.createNestApplication();
+		await app.init();
+	});
+
+	it("/ (GET)", () =>
+	{
+		return request(app.getHttpServer())
+			.get("/")
+			.expect(200)
+			.expect("Hello World!");
+	});
+});

```

Comment:
When do we add `;` and when don't we ? This seems to be some kind of a definition, not a call, so probably no use for a `;` here ?

Reply:
this file is automatically generated.
this statement :
	return request(app.getHttpServer())
		.get("/") // ; not added because we add it to another function of the pipeline
		.expect(200) // if we add ; here, we'll break the logic
		.expect("Hello World!"); // ; added at the end of the pipe

Reply:
The linter show us automatically when to use ;